src/ts/Conviva.d.ts

```diff
@@ -75,6 +75,9 @@ declare namespace Conviva {
     applicationName: string;
     assetName: string;
     custom: {};
+    /**
+     * @deprecated in at SDK 2.146.0.36444
```

Comment:
Should we keep this? 

Since we don't have to dependency to a specific conviva version i'm not sure.
Maybe we should create this dependency? Maybe just with a comment in the Readme, or a version check somewhere in the code and display a warning?

Reply:
If unused, then please remove

Reply:
Yes we need to introduce some kind of version management. As a first step, please add to the readme (somewhere on the top where it can be seen easily), which version(s) of the Conviva SDK this has been developed/tested for.

Reply:
Should we then also add a changelog so that we can say `Version x.y is compatible with Conviva SDK version a.b`?

Reply:
Since we don't have versioning yet you can add a changelog with git commit hashes in the meantime. That would be nice.